web/client/epics/catalog.js

```diff
@@ -117,7 +117,7 @@ export default (API) => ({
             .filter(({ layers, sources }) => isArray(layers) && isArray(sources) && layers.length && layers.length === sources.length)
             // maxRecords is 4 (by default), but there can be a possibility that the record desired is not among
             // the results. In that case a more detailed search with full record name can be helpful
-            .switchMap(({ layers, sources, options, startPosition = 1, maxRecords = 4 }) => {
+            .switchMap(({ layers, sources, filters, options, startPosition = 1, maxRecords = 4 }) => {
```

Comment:
can you change filters to be params?

Reply:
Updated code to use existing "options" argument.